Azure-Samples/chat-with-your-data-solution-accelerator
# refactor: Make document chunking pep8 compliant

## Purpose

* Make document chunking pep8 compliant. 
* Resolves #854 

## Does this introduce a breaking change?




## Pull Request Type
What kind of change does this Pull Request introduce?



- [ ] Bugfix (`fix`)
- [ ] Feature (`feat`)
- [x] Code style update (formatting, local variables) (`style`)
- [x] Refactoring (no functional changes, no api changes) (`refactor`)
- [ ] Documentation content changes (`docs`)
- [ ] Other... Please describe: (`perf`, `test`, `build`, `ci`, `chore`, `revert`)


## How to Test



* Run the tests:

```
poetry run pytest -m "not azure and not functional"    
poetry run pytest -m "functional"  
```

* Run the application locally and go to http://localhost:8502/Ingest_Data
* Upload a document on the above endpoint
* Verify the chunks are created on http://localhost:8502/Explore_Data

![SCR-20240509-ncja](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/4739534/e149e352-2149-481e-a0b6-5ca72667ee15)




## What to Check
Verify that the following are valid
* Verify that the chunks were created
